go/fuzz/pgp_key_read/fuzz.go

```diff
@@ -8,7 +8,7 @@ package libkb
 import "github.com/keybase/client/go/libkb"
 
 func Fuzz(data []byte) int {
-	key, err := libkb.ReadOneKeyFromBytes(data)
+	key, err := libkb.ReadOneKeyFromBytes(nil, data)
```

Comment:
My guess is that Max would say to never add a use of G, but I'll put it in there if you guys want.